Tools/CISupport/ews-build/steps.py

```diff
@@ -3130,7 +3130,9 @@ class CompileWebKit(shell.Compile, AddToLogMixin):
     env = {'MFLAGS': ''}
     warningPattern = '.*arning: .*'
     haltOnFailure = False
-    command = ['perl', 'Tools/Scripts/build-webkit', WithProperties('--%(configuration)s')]
+    build_command = ['perl', 'Tools/Scripts/build-webkit']
```

Comment:
any specific reason to remove "WithProperties('--%(configuration)s')" and add it later inside start()?

Reply:
This command gets joined instead passed as a string to the build command, so we have to resolve `WithProperties` before we pass the command to our parent class.